tools/bqman/golang/executionmode/executionmode.go

```diff
@@ -0,0 +1,79 @@
+package executionmode
```

Comment:
This file is missing the licensing header

Reply:
Done